src/pages/CubeListPage.js

```diff
@@ -53,7 +53,7 @@ const CubeListPageRaw = () => {
                     list: <ListView cards={boardcards} />,
                   }[cubeView]
                 }
-                {boardname === 'maybeboard' && <hr />}
+                {boardname !== 'mainboard' && <hr />}
```

Comment:
This is purely a stylistic change to mirror the check from line 41 and anticipate the existence of other boards.